imports/ui/pages/resources/resourceKindComponents.jsx

```diff
@@ -0,0 +1,261 @@
+import React from 'react';
+import moment from 'moment';
```

Comment:
There are some imports in this file that don't appear to be used.  `moment`, `ResourcesSingle` and `Blaze`